_book/04-advanced-topics/04-controlling-the-svg-output.md

```diff
@@ -1,11 +1,128 @@
+
+| --- | --- | --- | --- |
+| MEI unit | Raster | Staff size in mm | Example use |
+| --- | --- | --- | --- |
+| 11.5 | 0 | 9.2 | Educational music |
+| 9.875 | 1 | 7.9 | |
+| 9.25 | 2 | 7.4 | Piano music |
+| 8.75 | 3 | 7.0 | Single-staff parts
+| 8.125 | 4 | 6.5 | |
+| 7.5 | 5 | 6.0 | |
+| 6.875 | 6 | 5.5 | Choral music |
+| 6.0 | 7 | 4.8 | |
+| 4.625 | 8 | 3.7 | Full score |
+{: .table .table-condensed .table-sm .text-xsmall}
+
+{% aside .warning %}
+Up to Verovio 3.13 only integer MEI units are supported.
+{% endaside %}
+
+The example below shows the same file as above with the default A4 page size but with a unit value of `6.0`. More music will be rendered on a page because the staff size is smaller.
+
+{% include music-notation-only example="02-units" %}
+
+Changing the page dimension will increase the amount of music that fits on the page. The example below has been rendered with the default unit of `9.0` but a page height of `3050` and a page width of `2290`, a more typical paper size for sheet music than A4.
+
+{% include music-notation-only example="03-page-size" %}
+
+### Scaling
+
+The SVG output in Verovio can be scaled by using the `--scale` option. The option value is an integer representing a scaling percentage. It is `100` percent by default.
+
+When changing the scale option, Verovio will by default change the size of the output SVG image. For example, with the default page size and a scale option set to `50` percent, the resulting SVG image will have a size of **1485px** by **1050px**. The same amount of music will be engraved on the page as with the default scale value.
+
+In responsive environments, Verovio can be used to create user interfaces where the user can zoom in and out. This can be achieved by changing the scale and the page dimensions. Zooming out means increasing the page dimensions and reducing the scale by the same factor, and zooming in the opposite. For example, if the window in which the output of Verovio will be displayed is 1800px by 800px, these can be set as a page height and page width and Verovio will produce an SVG image that fits the window with the default scale value of 100 percent. To implement a zooming out function, for example by a factor 2, the page dimensions have to be changed to 3600 by 1600 and the scale to 50. The output SVG image will then still have a size that fits the window.
+
+Verovio has a `--scale-to-page-size` option that simplifies this process and that is recommended to be turned on in responsive environments. The advantage is that does not require the page dimensions to be calculated by the user when implementing zoom in or zoom out. With this option turned on, implementing a zoom function only requires the scale percentage to be changed and not the page dimensions. Keep in mind that the layout needs to be recalculated when the scale value is changed – see below.
```

Comment:
```suggestion
Verovio has a `--scale-to-page-size` option that simplifies this process. Using this option is recommended in responsive environments. The advantage is that does not require the page dimensions to be calculated directly, but Verovio will calculate changes to the page sizes internally depending on the scale value and an initial page size.
```

Reply:
I find it confusing to say the in changes the page sizes internally. The `--page-height` and `--page-width` values remain the same. Your wording might be understood that the values will change.

Reply:
From the previous paragraph you said that the "old way" means changing scale, width, and height in order to get the scale to work. 

The "new way" (with `--scale-to-page-size` enabled) needs to only adjust the scale value, so unless something different is happening I would guess that Verovio is automatically calculating the sizes internally. I was trying to draw a link between the previous paragraph and the "new" way.